src/utils/errors.ts

```diff
@@ -82,3 +82,13 @@ export interface ExecSyncError {
 export function instanceOfExecSyncError(object: any): object is ExecSyncError {
   return 'stderr' in object;
 }
+
+export function requireNonNullish<T>(x: T | null | undefined): T {
+  if (x === null) {
+    throw new Error('Unexpected null');
+  }
+  if (x === undefined) {
+    throw new Error('Unexpected undefined');
+  }
```

Comment:
Just out of curiosity, why not assert?

Reply:
This method is generic, so we don't know if `x` coming into it is an internal assumption (which deserves an assert and might be removed in the release version, automatically) or an assumption related to business logic (we can't ignore it). Thus, an error)